### DISCUSSION TITLE
To developers: Are you keeping DOCUMENTATION TO YOURSELVES Only?

### DISCUSSION THREAD

**Discussion by jmbxw2** [OpenSignLabs/OpenSign#982] [General]
It amazes me how little help you guys give on how to setup the platform.
Create user, how dbs works, etc. NO DOCUMENTATION.
Are you having second thoughts on this project being open source? it seems like.

**Comment by andrew-opensignlabs** [comment]
Have you checked https://docs.opensignlabs.com? User management is a newly released feature, and you can find the documentation [here](https://docs.opensignlabs.com/docs/help/Settings/manage-users/). Upgrade instructions specific to v2.1.0 can be found in release notes. Also, if you haven’t already, please join our Discord server for assistance with self-hosting.

**Reply by jmbxw2** [reply]
Thank you for your kind attitude towards my msg. I will check and re-check all of this and see if I come back with more passive aggressive msgs or not. Thanks again.
